modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/toolbar/contributor/BasePortletToolbarContributor.java

```diff
@@ -98,9 +99,7 @@ public Folder getFolder(
 		return folder;
 	}
 
-	public Boolean isShowActionsEnabled(
-		ThemeDisplay themeDisplay, PortletRequest portletRequest) {
-
+	protected Boolean isShowActionsEnabled(ThemeDisplay themeDisplay) {
```

Comment:
@MrJohn1911 This method doesn't need to return a `Boolean`, as `boolean` is enough. Please change it, thanks.